### PR TITLE
DDP-4423: Modify build of Angular builds to include sourcemaps in dev and test

### PR DESCRIPTION
**Pull request by fat23cat**
Looks like source maps don't affect app performance:
Without source maps:
Screenshot_2
With source maps:
Screenshot_1

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -121,7 +126,6 @@ commands:
       study_key:
         type: string
     steps:
-      - set-deployment-environment
```
Comment (fat23cat) [inline .circleci/config.yml:124]: Looks like when circle ci executed `deploy-stored-build`, `set-deployment-environment` was called twice: in the `deploy-stored-build` and then in the `deploy-exploded-build`.

File: .circleci/config.yml [hunk 2]
```diff
@@ -49,6 +49,7 @@ commands:
       - unless:
           condition: << parameters.skip_build >>
           steps:
+          - set-deployment-environment
```
Comment (fat23cat) [inline .circleci/config.yml]: We should determine working environment before building an app.

**Review by MocanaAtBroad — APPROVED** [review]
Was able to set breakpoint and bring up debugger on dev. Woo hoo!

**Comment by MocanaAtBroad** [comment]
@fat23cat : was looking at your performance measurement screenshot. Looked like we can save a second (or more) by turning on text compression. Do you know what they are referring to and how we could go about it? A GAE static resources config thing?

**Comment by fat23cat** [comment]
> @fat23cat : was looking at your performance measurement screenshot. Looked like we can save a second (or more) by turning on text compression. Do you know what they are referring to and how we could go about it? A GAE static resources config thing?

I think this feature should be enabled on the tcell side. https://web.dev/uses-text-compression/?utm_source=lighthouse&utm_medium=devtools

**Pull request merged**